repositories/QuestionnaireRepository.test.js

```diff
@@ -1,8 +1,4 @@
-const knex = require("../db/index");
-
-const createQuestionnaireTable = require("../migrations/20170620163533_create_questionnaires_table");
-const addSummaryColumn = require("../migrations/20171116131851_add_summary_column");
-const addCreatedByColumn = require("../migrations/20171211100317_add_created_by_to_questionnaires");
+const knex = require("../db");
 const QuestionnaireRepository = require("../repositories/QuestionnaireRepository");
 
 const buildQuestionnaire = (json = {}) => {
```

Comment:
Not necessarily something that needs to be done for this PR. But we have this buildQuestionnaire logic in a few tests. Might be worth refactoring it out into some kind of testUtils file.